composer.json

```diff
@@ -11,23 +11,25 @@
         }
     ],
     "require": {
-        "php": ">=5.5",
-        "behat/behat": "~3.0",
-        "psr/http-message": "~1.0"
+        "php": "^5.6||^7.0",
+        "behat/behat": "^3.0.6",
+        "psr/http-message": "^1.0",
+        "php-http/client-implementation": "^1.0"
     },
     "require-dev": {
-        "phpspec/phpspec": "~2.0",
-        "phpunit/phpunit": "~4.6",
-        "guzzlehttp/guzzle": "~6.0",
-        "kriswallsmith/buzz": "~0.10",
-        "symfony/process": "~2.3",
-        "symfony/filesystem": "~2.3"
-    },
-    "suggest": {
-        "guzzle/guzzle": "for guzzle integration",
-        "kriswallsmith/buzz": "for buzz integration"
+        "phpunit/phpunit": "^5.0",
+        "php-http/guzzle6-adapter": "^1.0",
+        "symfony/process": "^3.0",
+        "symfony/filesystem": "^3.0",
+        "php-http/discovery": "^0.8.0",
+        "puli/composer-plugin": "^1.0.0-beta9"
     },
     "autoload": {
         "psr-4": { "Zalas\\Behat\\RestExtension\\": "src/" }
-    }
+    },
+    "autoload-dev": {
+        "psr-4": { "Zalas\\Behat\\RestExtension\\": "tests/" }
+    },
+    "prefer-stable": true,
+    "minimum-stability": "beta"
```

Comment:
It is as `puli/composer-plugin` depends on `puli/repository` which is in beta too :)